【マークアップ】ユーザー新規登録/ログインページ

TomInJpn opened the pull request:
## what
deviseによるsign up in 機能実装（除くaddress）

## why
ユーザー機能実装のため

TomInJpn committed:
gemファイル変更

devise
haml
追加

TomInJpn committed:
ユーザー登録機能のみ実装

adressTable未実装
登録画面、ログイン画面、共にerr表示未実装
テスト未実施
jQuery衝突回避未実装
turbolinks無効

TomInJpn committed:
ユーザー登録機能終了

adressTable未実装
アドレス画面、ログイン画面、共に未実装
jQuery衝突回避実装
テスト未実施

TomInJpn committed:
ユーザーログイン機能終了

addressTable、address画面未実装
ログイン画面実装済
テスト未実施

shingoteshima commented on app/views/items/index.html.erb:
```diff
@@ -1,2 +1,10 @@
 <h1>Items#index</h1>
 <p>Find me in app/views/items/index.html.erb</p>
+
+<% if user_signed_in? %>
```
erbをhamlに変換しましょう。
他のファイルも同様です。

TomInJpn replied:
views/users x2
views/items
views/layouts
関連ファイルをhamlに変換しました。

shingoteshima commented:
コンフリクトの解消をお願いいたします。

TomInJpn committed:
ユーザー登録直後にログイン

ウィザード形式のため

TomInJpn committed:
ユーザー関連ビューのhaml化済

TomInJpn committed:
ユーザー関連ビューのhaml修正

ハッシュロケット修正

TomInJpn committed:
送付先情報入力画面フロント実装終了

registrations_controller.rbについて

resource.remember_me = true
sign_in :user, resource

を消し、
sign_up直後に
sign_inすることを取りやめています。

これは最終課題の条件にそぐわないためで、技術的に問題があることではないことをここに記します。

TomInJpn committed:
user・addressのview内から余分なbodyタグを削除

TomInJpn committed:
ユーザー登録機能wizard化

addressTable、address画面実装済
アドレスページのバリデーション未実装
テスト未実施

TomInJpn committed:
birthday統合

birth_year,month,dayをbirthdayに統合

理由
誕生日は3つ合わせて1つのデータであり分ける意味がない。
カラム数増により複雑になることを防ぐため。

アドレスページのバリデーション未実装
都道府県のハッシュタグ化未実装
テスト未実施

TomInJpn committed:
TopPageのdeployを受けてuser関連ページデザイン等変更

<header><footer>
は各担当が持つ。

またbackgroundのcolorは指定無し
※position:sticky;に初挑戦しています。

TomInJpn committed:
リファクタリング

TomInJpn committed:
Merge branch 'master' into devise実装

TomInJpn committed:
top重複ミス修正 items.scssをrename application.scss修正

TomInJpn committed:
sessionにrender適応　そのために_header修正　renderからpartial削除　デザイン修正

shingoteshima commented:
修正ありがとうございます！
LGTMです！！
お疲れさまでした！
